### PR TITLE
Vega avatar2

### PR DESCRIPTION
**Pull request by ddvega**
User can add an image(avatar) and make updates to username, email or password.

### CODE REVIEW

File: client/components/LoadUser.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  const { currentUser } = useAuth();
+  const api = useAPI();
+
+  const getUser = () => {
+    try {
+      api.get(`/users/${currentUser.uid}`).then(({ data }) => {
+        localStorage.setItem('picture', data.picture);
+        localStorage.setItem('username', data.username);
+        localStorage.setItem('email', data.email);
+      });
```
Comment (dlimx) [inline client/components/LoadUser.jsx:15]: Strong recommendation to use Context/Redux here

File: client/components/LoadUser.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      localStorage.getItem('email') === 'undefined' ||
+      !localStorage.getItem('picture') ||
+      !localStorage.getItem('username') ||
+      !localStorage.getItem('email')
+    ) {
+      localStorage.clear();
+      getUser();
+    }
+  });
+  return <></>;
```
Comment (dlimx) [inline client/components/LoadUser.jsx:34]: You can also just return `null`

**Review by dlimx — APPROVED** [review]
Looks good, minor comments

File: client/components/LoadUser.jsx (hunk 1 above)
Comment (ddvega) [reply client/components/LoadUser.jsx:15]: You know I actually did that and got rid of it. So you mean to have like a `userContext.jsx` file with a provider that wraps the components it affects as well as a `useAuth`? Like we have for `authContext.jsx`?

**Pull request merged**